app/src/main/java/com/manna/presentation/settings/FeedbackCategory.kt

```diff
@@ -1,7 +1,7 @@
 package com.manna.presentation.settings
 
-data class FeedbackCategory(
-    val category: String,
-    var click: Boolean,
-    var onClick: (FeedbackCategory) -> Unit
-)
\ No newline at end of file
+object FeedbackCategory{
+    const val ERROR = "error"
+    const val FEEDBACK = "feedback"
+    const val INQUIRY = "inquiry"
+}
```

Comment:
요거는 enum이 관리하기 편하지 않을까? 싶네유 👀 
```suggestion
enum FeedbackCategory(val blabla: String){
    ERROR("error"),
    FEEDBACK("feedback")
    INQUIRY("inquiry")
}
```